notebooks/NIRSpec_NSClean/FS_NSClean_example.ipynb

```diff
@@ -0,0 +1,1764 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #29.            Define a lower wavelengh end for the 1D spectrum. Helpful for BOTS data.
</pre><p>There is a small typo in the word "wavelength".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jdat_notebooks/pull/219/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Fixed